editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -448,6 +569,13 @@ EditorPlugin::AfterGUIInput Path3DEditorPlugin::forward_3d_gui_input(Camera3D *p
 					ur->add_undo_method(c.ptr(), "set_point_in", i, c->get_point_in(i));
 					ur->commit_action();
 					return EditorPlugin::AFTER_GUI_INPUT_STOP;
+				} else if (dist_to_p_up < click_dist && c->is_up_vector_enabled()) {
+					Ref<EditorUndoRedoManager> &ur = EditorNode::get_undo_redo();
```

Comment:
This needs to be changed to EditorUndoRedoManager *ur = EditorUndoRedoManager::get_singleton(); to compile with latest mainline.

Reply:
@ademan Bump :slightly_smiling_face: